src/database/migrations/20241216110839-update-enitityType-report-filter.js

```diff
@@ -0,0 +1,140 @@
+'use strict'
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		const transaction = await queryInterface.sequelize.transaction()
+		try {
+			const replacements = { report_filter: true }
+
+			const entityTypesToUpdate = [
+				{ value: 'medium', label: 'Medium' },
+				{ value: 'type', label: 'Type' },
+				{ value: 'recommended_for', label: 'Recommended For' },
+				{ value: 'categories', label: 'Categories' },
+			]
+
+			// Fetch and update records for each value/label pair
+			const updateEntityTypePromises = entityTypesToUpdate.map(async (item) => {
+				try {
+					const entityTypes = await queryInterface.sequelize.query(
+						`SELECT id, value, label, organization_id, model_names 
```

Comment:
@sumanvpacewisdom is it need to select these columns for verifying the entity type is exist or not ?